Software/backend/rest_api/Node.js/README.md

```diff
@@ -1,17 +1,122 @@
-<<<<<<< Updated upstream
-<!-- press ctrl+K then V to open a preview of the MarkDown file  -->
-=======
->>>>>>> Stashed changes
-# Page Main Title 
-short description of the directory's subject
+# Node.js and REST API 
+Here you will find information about REST API, Node.js servers and even some useful tool to become a professional coder.
 
-# Table Of Contents
+## Table Of Contents
 
-* [subject](#subject) 
+* [What is API](#What-is-API?) 
 
-* [subject2](#subject2) 
+* [What is REST](#What-is-REST)
 
-# Subject
-* [outer resource](http://...)  
-short description if required
-* [local resource](./resourceName.someFile)
\ No newline at end of file
+* [HTTP Methods - CRUD](#HTTP-Methods---CRUD)
+
+* [Node.js](#Node.js)
```

Comment:
this link doesnt seem to work, probably because it thinks the "." is part of the url. maybe just Node js with no .